True
torch diagonostics
2.5.1
12.4
True
start to load light curves
loaded files
finished loading lcs
3
Beginning training...
Traceback (most recent call last):
  File "/orcd/home/002/pdong/Astro UROP/trainmodelmulti.py", line 330, in <module>
    for batch in train_loader:
  File "/home/pdong/.conda/envs/myenv/lib/python3.11/site-packages/torch/utils/data/dataloader.py", line 701, in __next__
    data = self._next_data()
           ^^^^^^^^^^^^^^^^^
  File "/home/pdong/.conda/envs/myenv/lib/python3.11/site-packages/torch/utils/data/dataloader.py", line 757, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pdong/.conda/envs/myenv/lib/python3.11/site-packages/torch/utils/data/_utils/fetch.py", line 55, in fetch
    return self.collate_fn(data)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/orcd/home/002/pdong/Astro UROP/trainmodelmulti.py", line 282, in collate_fn_err_mult
    rate_low = [torch.tensor(lc['RATE'].values.byteswap().newbyteorder(), dtype=torch.float32) for lc[0] in batch]
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/orcd/home/002/pdong/Astro UROP/trainmodelmulti.py", line 282, in <listcomp>
    rate_low = [torch.tensor(lc['RATE'].values.byteswap().newbyteorder(), dtype=torch.float32) for lc[0] in batch]
                                                                                                   ^^
NameError: name 'lc' is not defined
